packages/calcite-components/src/components/shell/shell.stories.ts

```diff
@@ -842,6 +848,30 @@ export const slottedPanelTop_TestOnly = (): string =>
   </calcite-shell>
 `);
 
+export const contentBehindPanelBottom_TestOnly = (): string =>
```

Comment:
nitpick: I don't think `_TestOnly` is needed any longer. right @jcfranco?

Reply:
Correct. https://github.com/Esri/calcite-design-system/issues/9085 for context.